src/app/components/layout-sidebar/layout-sidebar-demo.component.html

```diff
@@ -1,3 +1,3 @@
-<stache-layout-sidebar>
+<stache layout="sidebar">
```

Comment:
You're probably right. We'd essentially be requiring folks to use `<stache>` around everything, even for custom layouts.